src/datasources/config-api/entities/chain.entity.ts

```diff
@@ -1,9 +0,0 @@
-import { NativeCurrency } from './native.currency.entity';
-
-export interface Chain {
-  chainId: string;
-  chainName: string;
-  transactionService: string;
-  vpcTransactionService: string;
-  nativeCurrency: NativeCurrency;
-}
```

Comment:
Why was this moved out of `datasource`?

Reply:
In [c8e5080](https://github.com/5afe/safe-client-gateway-nest/commit/c8e5080c1d7a313fc6a1b7d32441120abd78cc38) I moved all the schemas and also the shared entities to `datasources` 👍